src/pywrapper.cpp

```diff
@@ -248,14 +248,14 @@ bool PyWrapper::convert(void* in_, MultiTypeValue& out)
         return true;
     }
 
-    if (PyList_Check(in)) {
+    if (PySequence_Check(in)) {
         out.type = MultiTypeValue::Type::NONE;
         out.vi.clear();
         out.vf.clear();
         out.vs.clear();
 
-        for (Py_ssize_t i = 0; i < PyList_Size(in); i++) {
-            PyObject* el = PyList_GetItem(in, i);
```

Comment:
As for speed on could consider using PySequence_GETITEM. 

It has been checked already that index is within range.